app/views/layouts/application.html.erb

```diff
@@ -12,7 +12,8 @@
 
   <body class="bg-gradient-dark">
     <%= render 'layouts/message' %>
-    <% if [new_user_session_path, new_user_registration_path, user_registration_path, resume_path(1)].include? request.path %>
+    <% if [new_user_session_path, new_user_registration_path, user_registration_path,
+           resume_path(1), "/404", "/500", not_found_path, internal_server_error_path].include? request.path %>
```

Comment:
seriously?? 😮 
you will add all the path here manually throughout ?
look for alternate solution please :)